src/controllers/action-queue/types.ts

```diff
@@ -79,6 +79,7 @@ export interface SwapActionOp {
   fromTokenId?: string
   fromWalletId: string
   nativeAmount: string
+  payoutNativeAmount?: string
```

Comment:
Rename to `expectedPayoutNativeAmount`, move it to the end of the type's fields and add a comment explaining to the user when they should use this field.